config-single.toml

```diff
@@ -0,0 +1,19 @@
+p2p_port = 0
+
+[[nodes]]
+json_rpc_port = 4201
+eth_chain_id = 0x8001
+consensus.consensus_timeout = { secs = 5, nanos = 0 }
+consensus.genesis_committee = [ ["b27aebb3b54effd7af87c4a064a711554ee0f3f5abf56ca910b46422f2b21603bc383d42eb3b927c4c3b0b8381ca30a3", "12D3KooWESMZ2ttSxDwjfnNe23sHCqsJf6sNEKwgHkdgtCHDsbWU"] ]
```

Comment:
Side question - What is `genesis_committee` used for now that we've moved the committee into `deposit.sol`? I would assume that `genesis_deposits` makes it redundant?